scss/_buttons.scss

```diff
@@ -92,11 +92,11 @@
 
   .counter {
     color: #60b044;
-    background-color: #fff;
+    background-color: $bg-white;
   }
 
   &:hover {
-    color: #fff;
+    color: $text-white;
     background-color: darken(#60b044, 5%);
     background-image: linear-gradient(darken(#8add6d, 5%), darken(#60b044, 5%));
```

Comment:
> Another option to clean up the css is to apply a transparent gradient on top of the bg color, so that you don't need to be adding specific hex codes into the button gradients.

That's a really neat idea. Worth a separate pull request if you have the time? :pray: